src/pages/training.js

```diff
@@ -1,15 +1,324 @@
 import React from 'react';
 import Helmet from 'react-helmet';
+import withStyles from 'react-jss';
+import cx from 'classnames';
 
-import { Header } from '../components';
+import letsChat from '../assets/images/footer/lets_chat.png';
+import clients from '../data/clients.json';
+import imagePatternDots from '../assets/images/patterns/dots.png';
+import comprehensive from '../assets/images/training/comprehensive.png';
+import bestPractices from '../assets/images/training/best_practices.png';
+import realWorld from '../assets/images/training/real_world.png';
+import professionalInstructors from '../assets/images/training/professional_instructors.png';
+import { Header, Footer, Drops } from '../components';
+import { Contact, SignUp } from '../containers';
 
-const TrainingPage = () => (
+const styles = theme => ({
+  root: {
+    display: 'flex',
+    minHeight: '100vh',
+    flexDirection: 'column'
+  },
+
+  // MAIN
+  main: {
+    flex: '1',
+    '& h1': {
+      marginTop: 0,
+      fontSize: 20
+    }
+  },
+  mainContent: {
+    paddingTop: 20,
+    margin: [0, 20]
+  },
+  leftContent: {
+    maxWidth: '100%'
+  },
+  trainings: {
+    marginTop: 80
+  },
+  training: {
+    display: 'flex',
+    marginTop: 40
+  },
+  trainingImage: {
+    width: 60,
+    '& img': {
+      width: 60,
+      marginTop: 5
+    }
+  },
+  trainingContent: {
+    flex: 1,
+    paddingLeft: 20,
+    '& h2': {
+      marginTop: 0,
+      fontSize: 16,
+      marginBottom: 10
+    },
+    '& p': {
+      fontSize: 14
```

Comment:
Don't really know if this is necessary. Please check it out in the designs. Since we are adding this kind of content in many places.

Reply:
effectively i reviewed this and it is necessary to do, to approximate to the design